lib/libutee/include/tee_api_defines.h

```diff
@@ -260,8 +260,170 @@
 #define TEE_ATTR_ECC_PRIVATE_VALUE          0xC0000341
 #define TEE_ATTR_ECC_CURVE                  0xF0000441
 
-#define TEE_ATTR_BIT_PROTECTED		    (1 << 28)
-#define TEE_ATTR_BIT_VALUE		    (1 << 29)
+#define TEE_ATTR_BIT_PROTECTED              (1 << 28)
+#define TEE_ATTR_BIT_VALUE                  (1 << 29)
+
+/* Panicked Functions Identification */
```

Comment:
Panicked IDs are defined in the specification. Implementation usage is not clearly defined but recommendations can be found (TEE TA Debug Specification or or an implementation-specific alternative). I would prefer to keep them but I am OK to wait with these defines until they are used if the majority of reviewer thinks it OK to remove them.